cirq/optimizers/eject_z_test.py

```diff
@@ -12,17 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from typing import Iterable
-
 import cirq
 from cirq.optimizers.eject_z import _try_get_known_z_half_turns
 
 
 def assert_optimizes(before: cirq.Circuit,
-                     expected: cirq.Circuit,
-                     post_opts: Iterable[cirq.OptimizationPass] = (
-                             cirq.DropEmptyMoments(),
-                     )):
+                     expected: cirq.Circuit):
```

Comment:
Oh. DropEmptyMoments() is the solution. Will add in next iteration.

Reply:
Update: Putting
```
  followup_optimizations = [
        cirq.DropNegligible(),
        cirq.DropEmptyMoments()
    ]
    for post in followup_optimizations:
        post.optimize_circuit(before)
        post.optimize_circuit(expected) 
```
Hans't gotten rid of the quirks that required i change the tests.